Source/OxyPlot.Wpf.Shared/PlotViewBase.cs

```diff
@@ -454,6 +455,12 @@ private bool IsInVisualTree()
                     return true;
                 }
 
+                //Check if the parent is an AdornerDecorator like in an ElementHost
+                if (dpObject is AdornerDecorator)
+                {
+                    return true;
+                }
```

Comment:
What scenario does this cover? (I've hardly used WPF in years, and not familiar with adorners)

Reply:
This is the parent of the ElementHost. It is used to wrap WPF in Windows Forms